ArtOfIllusion/src/artofillusion/CreatePolygonTool.java

```diff
@@ -66,155 +71,220 @@ public String getToolTipText()
     return Translate.text("createPolygonTool.tipText");
   }
 
-  void tabulateSines()
+  private void tabulateSines()
   {
-    int i;
-
     sine = new double [sides];
     cosine = new double [sides];
     minsine = mincosine = 1.0;
     maxsine = maxcosine = -1.0;
-    for (i = 0; i < sides; i++)
+    for (int i = 0; i < sides; i++)
     {
-      sine[i] = Math.sin((i+0.5)*2.0*Math.PI/sides);
-      sine[i] = 1e-10*Math.round(sine[i]*1e10);
+      sine[i]   = Math.sin((i+0.5)*2.0*Math.PI/sides);
+      sine[i]   = 1e-10*(Math.round(sine[i]*1e10));
```

Comment:
Only thing I've done so far: you've got a redundant pair of parentheses here.

Reply:
Line 83

Reply:
Oops. I tried somethig different here, this happened at returning the original.